RocketChat/Rocket.Chat.ReactNative
# Chore: Update icon set

## Proposed changes

- Updates directory icon to the new look and adds a few directory/federation icons
- Adds emoji picker new icons

### Federation 
![image](https://user-images.githubusercontent.com/804994/178343785-0c1cf181-b470-422e-98cf-da3e663742b5.png)

### Directory
![image](https://user-images.githubusercontent.com/804994/178343693-ffb64460-487e-4765-aa4b-081bdd7cf43f.png)

### Emoji Picker
![image](https://user-images.githubusercontent.com/804994/178584189-3175275e-8a31-4572-924e-6d850ea042fd.png)


## Issue(s)	


## How to test or reproduce

- The only visual change should be the new directory icon on rooms list header right section
  - It used to be the `globe`, which is now the federation icon, and now is the notebook with the hashtag inside.
- This change updates the whole icon set, so a visual regression of all icons is needed

## Screenshots

## Types of changes




## Checklist



## Further comments
